lightwood/shared/helpers.py

```diff
@@ -0,0 +1,16 @@
+import torch
```

Comment:
i think the folder should be could lightwood/helpers/
rather than 'shared' and this file should be called device.py

Reply:
Hmh, yeah, I think that might be a more consistent naming schema, will change.